### ISSUE TITLE
pyftsubset --flavor=woff2 regression in 3.43.0: AttributeError: WOFFFlavorData instance has no attribute 'transformedTables'

### ISSUE THREAD

**Issue by papandreou** [fonttools/fonttools#1650]
After my CI builds started picking up fonttools 3.43.0 instead of 3.42.0, it's getting these errors: https://travis-ci.org/assetgraph/assetgraph/jobs/547453874#L645-L675

Seems like it's reproducible with:

```
$ pip install brotli fonttools==3.43.0 zopfli
$ pyftsubset Roboto-Regular.woff --output-file=foo.woff2 --flavor=woff2 --text=abc
WARNING: FFTM NOT subset; don't know how to subset; dropped
Traceback (most recent call last):
  File "/home/andreas/.local/bin/pyftsubset", line 11, in <module>
    sys.exit(main())
  File "/home/andreas/.local/lib/python2.7/site-packages/fontTools/misc/loggingTools.py", line 375, in wrapper
    return func(*args, **kwds)
  File "/home/andreas/.local/lib/python2.7/site-packages/fontTools/subset/__init__.py", line 2876, in main
    save_font(font, outfile, options)
  File "/home/andreas/.local/lib/python2.7/site-packages/fontTools/misc/loggingTools.py", line 375, in wrapper
    return func(*args, **kwds)
  File "/home/andreas/.local/lib/python2.7/site-packages/fontTools/subset/__init__.py", line 2723, in save_font
    font.save(outfile, reorderTables=options.canonical_order)
  File "/home/andreas/.local/lib/python2.7/site-packages/fontTools/ttLib/ttFont.py", line 173, in save
    writer_reordersTables = self._save(tmp)
  File "/home/andreas/.local/lib/python2.7/site-packages/fontTools/ttLib/ttFont.py", line 214, in _save
    writer.close()
  File "/home/andreas/.local/lib/python2.7/site-packages/fontTools/ttLib/woff2.py", line 230, in close
    if isTrueType and "glyf" in self.flavorData.transformedTables:
AttributeError: WOFFFlavorData instance has no attribute 'transformedTables'
```

Whereas with 3.42.0 it works:

```
$ pip install brotli fonttools==3.42.0 zopfli
$ pyftsubset Roboto-Regular.woff --output-file=foo.woff2 --flavor=woff2 --text=abc
WARNING: FFTM NOT subset; don't know how to subset; dropped
$ ls -la foo.woff2 
-rw-r--r-- 1 andreas andreas 852 jun 19 00:54 foo.woff2
```

(`Roboto-Regular.woff` can be obtained from eg. https://github.com/grpc/grpc.github.io/blob/master/fonts/roboto/Roboto-Regular.woff)

**Comment by anthrotype** [comment]
Thanks for reporting. I'll make a bugfix release today, in the meantime please pin the version to 3.42.0.

**Comment by anthrotype** [comment]
@papandreou FYI I've just released 3.43.1 with the fix.

**Comment by papandreou** [comment]
@anthrotype, thanks for the fast action! Updating to `fonttools==3.43.1` fixed my build 🤗
